MicrosoftTeams.schema.json

```diff
@@ -248,6 +248,10 @@
             "$ref": "#/definitions/httpsUrl",
             "description": "The url which points to the entity UI to be displayed in the Teams canvas."
           },
+          "contentBotId": {
```

Comment:
Everywhere else in the manifest where we have bot powered capabilities this property is simply called "botId". Did you consider sticking with that pattern?

Reply:
We discussed this in the PRs thread for this PR:
https://teams.microsoft.com/l/message/19:ae80c4f9350f4faf925b2902f90d3070@thread.tacv2/1600706977056?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=f9ce4f7b-3bd8-4b25-a573-d484beea4ee0&parentMessageId=1600706977056&teamName=HEX%20Team&channelName=PRs&createdTime=1600706977056
![image](https://user-images.githubusercontent.com/43385844/93920669-577fbc00-fcc4-11ea-86f1-b781b37462ee.png)